src/core/models.js

```diff
@@ -0,0 +1,33 @@
+import seer from 'seer';
```

Comment:
>>Should the file name be seer-integration.js or similar?
> Nope, seer is what it should be

Of course, was not suggesting we change the module name. 

I was actually referring to name of the new file that this PR creates in luma.gl: `src/core/models.js`. 

I feel it would make more sense as something like `src/debug/seer-integration.js`